Gems/Terrain/Code/Source/TerrainRenderer/TerrainFeatureProcessor.h

```diff
@@ -125,17 +129,19 @@ namespace Terrain
             UseTextureHeight =     0b0000'0000'0000'0000'0000'0000'0010'0000,
             UseTextureSpecularF0 = 0b0000'0000'0000'0000'0000'0000'0100'0000,
 
-            FlipNormalX =          0b0000'0000'0000'0000'0000'0000'1000'0000,
-            FlipNormalY =          0b0000'0000'0000'0000'0000'0001'0000'0000,
+            FlipNormalX =          0b0000'0000'0000'0001'0000'0000'0000'0000,
+            FlipNormalY =          0b0000'0000'0000'0010'0000'0000'0000'0000,
 
-            BlendModeMask =        0b0000'0000'0000'0000'0000'0110'0000'0000,
+            BlendModeMask =        0b0000'0000'0000'1100'0000'0000'0000'0000,
```

Comment:
I went with binary because it's way easier to tell which bit is flipped vs hex. Unfortunate that we don't have support in azsl for binary types yet. I kept the binary value as a comment in the azsl for reference. So this was an intentional choice.